WalletWasabi/WabiSabi/Backend/WabiSabiConfig.cs

```diff
@@ -110,9 +110,8 @@ public WabiSabiConfig(string filePath) : base(filePath)
 	[JsonConverter(typeof(MoneyBtcJsonConverter))]
 	public Money MaxSuggestedAmountBase { get; set; } = Money.Coins(0.1m);
 
-	[DefaultValue(false)]
-	[JsonProperty(PropertyName = "IsCoinVerifierEnabled", DefaultValueHandling = DefaultValueHandling.Populate)]
-	public bool IsCoinVerifierEnabled { get; set; } = false;
+	[JsonProperty(PropertyName = "RiskFlags", DefaultValueHandling = DefaultValueHandling.Populate)]
+	public List<int>? RiskFlags { get; set; } = new();
```

Comment:
I think we need the nullability here, bc we cannot define a default value for the type `List<int>`.
This part of the config file is created like this `RiskFlags: null`, which breaks the app if we don't allow it to be nullable.

Thoughts?

Reply:
If it works, then it's good.